### PR TITLE
Add the Thycotic DevOps Secrets Vault lookup plugin.

### PR DESCRIPTION
**Pull request by amigus**
##### SUMMARY
Add a lookup plug-in capable of retrieving Secrets from Thycotic's DevOps Secrets Vault.  The plug-in uses python-dsv-sdk which is available here: https://pypi.org/project/python-dsv-sdk/

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

dsv

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/plugins/lookup/dsv.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# python 3 headers, required if submitting to Ansible
```
Comment (Akasurde) [inline lib/ansible/plugins/lookup/dsv.py:1]: @amigus Could you please submit this PR [here](https://github.com/ansible-collections/community.general)?

File: lib/ansible/plugins/lookup/dsv.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        self.set_options(var_options=variables, direct=kwargs)
+
+        vault_parameters = {
+            "tenant": self.get_option("tenant"),
+            "client_id": self.get_option("client_id"),
+            "client_secret": self.get_option("client_secret"),
+            "url_template": self.get_option("url_template"),
+        }
+
+        display.vvv("DevOps Secrets Vault parameters: %s" % vault_parameters)
```
Comment (Akasurde) [inline lib/ansible/plugins/lookup/dsv.py:108]: Do not display vault_parameters to stdout or logs since it contains client_secret. This will lead to CVE.

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: lib/ansible/plugins/lookup/dsv.py (hunk 1 above)
Comment (amigus) [reply lib/ansible/plugins/lookup/dsv.py:1]: Done. Sorry for my mistake.  Thanks!

File: lib/ansible/plugins/lookup/dsv.py (hunk 2 above)
Comment (amigus) [reply lib/ansible/plugins/lookup/dsv.py:108]: Done.

**Pull request closed without merging**